config/locales/en/course/assessment/question/voice_responses.yml

```diff
@@ -0,0 +1,16 @@
+en:
+  course:
+    assessment:
+      question:
+        voice_responses:
+          new:
+            voice_response_header: 'Voice Response Question'
```

Comment:
Can we use `Audio Response Question` instead of Voice Response? We can keep the backend name as it is, just change the front-end labels.

Reply:
OK, will fix this.

Reply:
I will still keep the key to be ```voice_response_header``` to keep it consistent with the backend name.